dhis-2/dhis-services/dhis-service-analytics/src/test/java/org/hisp/dhis/analytics/event/data/AbstractAnalyticsServiceTest.java

```diff
@@ -72,7 +73,7 @@
  * @author Luciano Fiandesio
  */
 @ExtendWith(MockitoExtension.class)
-class AbstractAnalyticsServiceTest {
+class AbstractAnalyticsServiceTest extends TestBase {
```

Comment:
I think it gets messy when mixing a mockito test with spring stuff. Maybe you can copy what you need instead or make a `MockitoTestBase`?